protocol/messenger_contacts.go

```diff
@@ -847,14 +847,14 @@ func (m *Messenger) blockContact(response *MessengerResponse, contactID string,
 		}
 
 		// We remove anything that's related to this contact request
-		notifications, err := m.persistence.DeleteChatContactRequestActivityCenterNotifications(contact.ID, m.getCurrentTimeInMillis())
+		updatedAt := m.GetCurrentTimeInMillis()
+		notifications, err := m.persistence.DeleteChatContactRequestActivityCenterNotifications(contact.ID, updatedAt)
```

Comment:
@qfrank similar to above, we don't need to delete a previous notification, but simply open a popup of that contact displaying it in the blocked state.

Reply:
I found this comment `// We remove anything that's related to this contact request` comes from @MishkaRogachev , WDYT about sale's comment? @MishkaRogachev

Reply:
I checked with @John-44 , in case of blocking contact, we have to delete all notifications associated with this contact.